.goreleaser.yml

```diff
@@ -0,0 +1,270 @@
+      - -X github.com/maticnetwork/heimdall/version.Commit={{.Commit}}
+      - -X github.com/cosmos/cosmos-sdk/version.Name=heimdall
+      - -X github.com/cosmos/cosmos-sdk/version.ServerName=heimdalld
+      - -X github.com/cosmos/cosmos-sdk/version.ClientName=heimdallcli
+      - -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}}
+      - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
+  
+  - <<: *xbuildbridge
+    id: bridge-darwin-arm64
+    goos:
+      - darwin
+    goarch:
+      - arm64
+    env:
+      - CC=oa64-clang
+      - CXX=oa64-clang++
+  
+  - <<: *xbuildbridge
+    id: bridge-linux-amd64
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CC=gcc
+      - CXX=g++
+    ldflags:
+      # We need to build a static binary because we are building in a glibc based system and running in a musl container
+      - -linkmode external -extldflags "-static"
+
+  - <<: *xbuildbridge
+    id: bridge-linux-arm64
+    goos:
+      - linux
+    goarch:
+      - arm64
+    env:
+      - CC=aarch64-linux-gnu-gcc
+      - CXX=aarch64-linux-gnu-g++
+    ldflags:
+      # We need to build a static binary because we are building in a glibc based system and running in a musl container
+      - -linkmode external -extldflags "-static"
+
+nfpms:
+  - vendor: 0xPolygon
+    homepage: https://polygon.technology
+    maintainer: Polygon Team <team@polygon.technology>
+    description: Polygon Blockchain
+    license: GPLv3 LGPLv3
+
+    formats:
+      - apk
+      - deb
+      - rpm
+
+    contents:
+      - src: builder/files/heimdalld.service
+        dst: /lib/systemd/system/heimdalld.service
+        type: config
+      - src: builder/files/heimdalld-rest-server.service
+        dst: /lib/systemd/system/heimdalld-rest-server.service
+        type: config
+      - src: builder/files/heimdalld-bridge.service
+        dst: /lib/systemd/system/heimdalld-bridge.service
+        type: config
+
+    overrides:
+      rpm:
+        replacements:
+          amd64: x86_64
+
+snapshot:
+  name_template: "{{ .Tag }}.next"
+
+dockers:
+  - image_templates:
+      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-amd64
+    dockerfile: Dockerfile.release
+    use: buildx
+    goarch: amd64
+    ids:
+      - heimdalld-linux-amd64
+      - heimdallcli-linux-amd64
+      - bridge-linux-amd64
+    build_flag_templates:
+      - --platform=linux/amd64
+    skip_push: false
+    extra_files:
+      - docker/entrypoint.sh
+  
+  - image_templates:
+      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-arm64
+    dockerfile: Dockerfile.release
+    use: buildx
+    goarch: arm64
+    ids:
+      - heimdalld-linux-arm64
+      - heimdallcli-linux-arm64
+      - bridge-linux-arm64
+    build_flag_templates:
+      - --platform=linux/arm64
+    skip_push: false
+    extra_files:
+      - docker/entrypoint.sh
+
+docker_manifests:
+  - name_template: 0xpolygon/{{ .ProjectName }}:{{ .Version }}
```

Comment:
Are we publishing in 0xPolygon docker org?

Reply:
That's the goal after the announcement and bump to v0.3.x, yes